opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AisStagesCommonUtil.java

```diff
@@ -50,14 +59,26 @@ public static RequestSpecification withDefaultHeaders(String fintechUserId) {
     }
 
     public static RequestSpecification withHeadersWithoutIpAddress(String fintechUserId) {
-        return RestAssured
-                .given()
-                    .header(AUTHORIZATION, DEFAULT_AUTHORIZATION)
-                    .header(BANK_ID, SANDBOX_BANK_ID)
-                    .header(FINTECH_REDIRECT_URL_OK, FINTECH_REDIR_OK)
-                    .header(FINTECH_REDIRECT_URL_NOK, FINTECH_REDIR_NOK)
-                    .header(FINTECH_USER_ID, fintechUserId)
-                    .header(SERVICE_SESSION_PASSWORD, SESSION_PASSWORD)
-                    .header(X_REQUEST_ID, UUID.randomUUID().toString());
+        RequestSpecification specification = RestAssured
```

Comment:
Please rename method withHeadersWithoutIpAddress to withSignedHeadersWithoutIpAddress

Reply:
Renamed